guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -8,7 +8,7 @@ ifdef::satellite[]
 endif::[]
 
 ifndef::satellite[]
-* If you plan to provision hosts from {Project} or {SmartProxy}, you must install {Project} and {SmartProxies} on a system with `grub2` version 2.05 or higher or system with fixes for HTTP Boot, for example CentOS version 7.9 or 8.3 or higher.
+* If you plan to provision hosts from {Project} or {SmartProxy}, you must install {Project} and {SmartProxies} on a system with `grub2` version 2.05 or higher or system with fixes for HTTP Boot.
```

Comment:
Given it was fixed in EL 8.3, can we safely assume this whole section is redundant?